app/services/idv/steps/ipp/address_step.rb

```diff
@@ -3,7 +3,26 @@ module Steps
     module Ipp
       class AddressStep < DocAuthBaseStep
         STEP_INDICATOR_STEP = :verify_info
-        def call; end
+
+        def call
+          Idv::Ipp::AddressForm::ATTRIBUTES.each do |attr|
+            flow_session[:pii_from_user][attr] = flow_params[attr]
+          end
+        end
+
+        def extra_view_variables
+          {
+            pii: flow_session[:pii_from_user],
+          }
+        end
```

Comment:
Is this view variable being used?

Reply:
Ah, you're right, I just removed all references to it. I'll remove it and add it back when we actually use it

Reply:
Removed in [256d01a](https://github.com/18F/identity-idp/pull/6500/commits/256d01adb04c79d28feb5af949d9de1998edcfaa)